docker/README.md

```diff
@@ -4,10 +4,10 @@ To simplify environment setup for the MosaicML `Composer` library, we provide a
 leverage.
 
 All Docker images come preinstalled with the following packages:
-* Ubuntu 18.04
-* Python 3.8.0
-* Nvidia CUDA 11.1.1
-* PyTorch 1.9.0
+* Ubuntu 20.04
+* Python
+* Pytorch
+* Nvidia CUDA (except if using a CPU-only flavor of pytorch)
```

Comment:
Let's document the matrix build here.  We are going to need to support this "legacy" image as well, also looks like we should have a Python 3.7 version of the image we can test with.

Reply:
Sg, I will put this in table form. It's only a handful of images now.

Reply:
Python 3.7 is addressed in #68. But I will update the docs here to reflect that.

Reply:
I will wait on adding in the build matrix until we have our Jenkins CI/CD configured and docker images published. Let's discuss which images we want to be "officially supported".